sdk/eventhub/azure-eventhub/README.md

```diff
@@ -115,8 +114,10 @@ The following sections provide several code snippets covering some of the most c
 - [Inspect an Event Hub](#inspect-an-event-hub)
 - [Publish events to an Event Hub](#publish-events-to-an-event-hub)
 - [Consume events from an Event Hub](#consume-events-from-an-event-hub)
+- [Consume events from an Event Hub in Batches](#consume-events-from-an-event-hub-in-batches)
```

Comment:
lower case "in batch" to align with the title of asynchronous one

Reply:
fixed